semgrep/semgrep/meta.py

```diff
@@ -202,7 +202,9 @@ def repo_name(self) -> str:
 
     @property
     def repo_url(self) -> Optional[str]:
-        if self.repo_name:
+        if "GITHUB_ENTERPRISE_URL" in os.environ:
+            return os.getenv("GITHUB_ENTERPRISE_URL")
+        elif self.repo_name:
             return f"https://github.com/{self.repo_name}"
```

Comment:
How about using the `GITHUB_SERVER_URL` environment variable (https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables) instead of introducing a new environment variable?
```suggestion
        if self.repo_name:
            github_url = os.getenv("GITHUB_SERVER_URL", "https://github.com")
            return f"{github_url}/{self.repo_name}"
```

Reply:
Sg!

Updated PR